misc/meeting-notes.md

```diff
@@ -1,3 +1,18 @@
+# 2019-03-06
+
+- parallel test harness runner landed!
+- we reviewed the content for creating a PR for the Grammar-WG to rust-lang/lang-team
+  - [PR created](https://github.com/rust-lang/lang-team/pull/3)
+- @CAD is still working on [the %% PR](https://github.com/rust-lang-nursery/gll/pull/2)
+
+- @qmx is taking unit-testing as his next task
```

Comment:
I believe the gist of our decision wrt. handling parse forests & ambiguity was:
- We have `$production(.$random_description?)(.$number_in_sequence)?.expected`.
- We test that all of the `.expected` files are in the parse forest.
- If `$number_in_sequence` is omitted then this asserts uniqueness (the forest is a tree).